File: 1281.cpp
/*
	Author: Zoljargal Gantumur
	Runtime: 0 ms, faster than 100.00% of C++ online submissions
	Memory Usage: 6.1 MB, less than 21.25% of C++ online submissions
*/

class Solution {
public:
    int subtractProductAndSum(int n) {
        int sum=0;
        int product = 1;
        while(n!=0){
            sum += n%10;
            product *= n%10;
            n = n/10;
        }
        return product-sum;
    }
};

File: 1281.py
# Author: Zoljargal Gantumur
# Runtime: 76 ms, faster than 6.48% of Python3 online submissions
# Memory Usage: 29.5 MB, less than 5.05% of Python3 online submissions

import numpy
class Solution:
    def subtractProductAndSum(self, n: int) -> int:
        lst = [int(i) for i in str(n)]
        return numpy.prod(lst) - sum(lst)
        
